apps/worker/src/app/workflow/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -247,9 +247,9 @@ export class SendMessageEmail extends SendMessageBase {
         const shouldDisableInlineCss = await this.getFeatureFlag.execute(
           GetFeatureFlagCommand.create({
             key: FeatureFlagsKeysEnum.IS_EMAIL_INLINE_CSS_DISABLED,
-            environmentId: command.environmentId,
+            environmentId: 'system',
             organizationId: command.organizationId,
-            userId: command.userId,
+            userId: 'system',
```

Comment:
Applying the same context as we have [here](https://github.com/novuhq/packages-enterprise/blob/next/billing/src/guards/quota-throttler.guard.ts#L65) for quota-throttler guard to attempt a fix at rule evaluation.